src/main/resources/io/jenkins/plugins/artifact_manager_s3/S3BlobStore/config.jelly

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:section title="${%Artifact Manager S3}">
+        <j:if test="${!descriptor.isPropertyConfigured()}">
+            <f:entry title="${%S3 Bucket Name}" field="container">
+                <f:textbox/>
+            </f:entry>
+            <f:entry title="${%S3 Bucket Region}" field="region">
+                <f:select/>
+            </f:entry>
+            <f:entry title="${%Base folder}" field="prefix">
+                <f:textbox/>
+            </f:entry>
+        </j:if>
+        <j:if test="${descriptor.isPropertyConfigured()}">
+            <strong>Configured by properties - Read Only</strong>
```

Comment:
Weird and unnecessary, delete IMO.

Reply:
IMHO this avoid to confused the used when the configuration is made with properties if you allow modifying the configuration from the UI when you use properties, Which configuration is the good one? when you restart which configuration you will use?

Reply:
I think that generally system properties are only used if the config is not set. So UI config takes precedence

Reply:
My suggestion is that this “way” be the same as for any other Jenkins feature: via JEP-201, or Groovy init hooks, or whatever is most appropriate for the installation technology.

The expectation for Jenkins Essentials is that when you run it on AWS, stuff will be preconfigured appropriately for you. But there is no solid proposal for that yet that I am aware of; the last conversation I remember with @rtyler, he seemed OK with the vague idea that when you install Jenkins from the AWS Marketplace, there will be some setup code which deals with AWS-specific configuration as well as more generic things like initial security, rather than there being magic autodetection of AWS from a universal image.